### ISSUE TITLE
xCAT probe has problems building RPM in local repo 

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#1549]
When running `buildcore.sh` in my local fork copy, xCAT probe cannot build the rpm:

```
chmod: cannot access '/home/vhu/rpmbuild/BUILDROOT/xCAT-probe-2.12.2-snap201607181006.x86_64//opt/xcat/probe/lib/perl/xCAT': Permission denied
chmod: cannot access '/home/vhu/rpmbuild/BUILDROOT/xCAT-probe-2.12.2-snap201607181006.x86_64//opt/xcat/probe/lib/perl/probe_utils.pm': Permission denied
error: Bad exit status from /var/tmp/rpm-tmp.hNNn8l (%install)
    Bad exit status from /var/tmp/rpm-tmp.hNNn8l (%install)
...
...
...
Error:  build of the following RPMs failed:  xCAT-probe

```

**Comment by whowutwut** [comment]
In addition, running the above build results in the following untracked file: 

```
[vhu@victorvm7 xcat-core]$ git status
...
# Untracked files:
#   (use "git add <file>..." to include in what will be committed)
#
#	xCAT-probe/lib/perl/xCAT/GlobalDef.pm
```

**Comment by hu-weihua** [comment]
@whowutwut , if we fix issue #1571, the ``Permission denied``  of the issue will gone.

Due to xCAT probe reuses some code of xCAT, but we don't hope to maintain 2 copies of these file. so we just copy these file to ``xCAT-probe`` directory when we do probe package. so there isn't ``xCAT/GlobalDef.pm`` under ``xCAT-probe/lib/perl/``.

**Comment by whowutwut** [comment]
After pulling in the code change from the pull request #1571 , building in my forked copy, I don't see this untracked file anymore.  Closing the issue for now .
